packages/kit/types/ambient.d.ts

```diff
@@ -233,9 +233,9 @@ declare module '$service-worker' {
 	 */
 	export const prerendered: string[];
 	/**
-	 * The result of calling `Date.now()` at build time. It's useful for generating unique cache names inside your service worker, so that a later deployment of your app can invalidate old caches.
+	 * See [`config.kit.version`](/docs/configuration#version). It's useful for generating unique cache names inside your service worker, so that a later deployment of your app can invalidate old caches.
```

Comment:
hmm. we may have broken scroll handling recently. When I go to https://kit.svelte.dev/docs/configuration#version it doesn't scroll to the correct location

Reply:
It works for me on Firefox. But yet, it does not work for me on Chrome.